Student/Student/Assignments/AssignmentDetails/AssignmentDetailsViewController.swift

```diff
@@ -228,7 +228,7 @@ class AssignmentDetailsViewController: UIViewController, AssignmentDetailsViewPr
         descriptionHeadingLabel?.text = quiz == nil
             ? NSLocalizedString("Description", bundle: .student, comment: "")
             : NSLocalizedString("Instructions", bundle: .student, comment: "")
-        descriptionView?.loadHTMLString(assignment.descriptionHTML, baseURL: baseURL)
+        descriptionView?.loadHTMLString(presenter?.assignmentDescription() ?? "", baseURL: baseURL)
```

Comment:
Why is the `?? ""` necessary? `assignmentDescription` should always return some string right?

Reply:
but the presenter is optional, i could do a guard for the presenter but ...meh...

Reply:
Ohh I missed that. Gotcha.